tasmoadmin/resources/js/Sonoff.js

```diff
@@ -261,7 +261,7 @@ class Sonoff {
     setDeviceValue(id, field, newvalue, td) {
         $.ajax({
             dataType: "json",
-            url: "?doAjax",
+            url: "/actions?doAjax",
             timeout: this.options.timeout * 1000,
             cache: false,
             type: "post",
```

Comment:
In this file it broke something in baseUrl

Reply:
Ahh forgot to consider the `base_url` here - should be an easy one to fix